docs/guides/integration-prebid.md

```diff
@@ -1,253 +1,105 @@
-
-When you download the Prebid.js package, add the UID2 module by checking the box next to the module named **Unified ID 2.0**, listed under the section **User ID Modules**.
-
-When you've added Prebid.js to your site and confirmed that it's working properly, you're ready to configure the UID2 module.
-
-:::tip
-To make sure that the UID2 module is installed, find the string `uid2IdSystem` in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::caution
+UID2 is not designed to be used where GDPR applies. The module checks the consent data that's passed in, and does not operate if the `gdprApplies` flag is set to `true`.
 :::
 
-## Configure the UID2 Module
-
-To configure the UID2 module, call `pbjs.setConfig` with an object containing the **public key** and **subscription ID** that you received during account setup, as well as the user's hashed or unhashed [DII](../ref-info/glossary-uid.md#gl-dii) (email address or phone number).
+## Generating the UID2 Token
 
-Once it's configured, the UID2 module generates a UID2 token for the user and stores it in the user's browser. The module automatically refreshes the token as required while your site is open in the user's browser.
+Depending on access to DII, there are two methods to generate UID2 tokens for use with Prebid.js, as shown in the following table.
 
-You can pass the user's DII to the UID2 module either hashed or unhashed. If you pass the DII unhashed, the UID2 module hashes it for you. If you want to pass the DII to the module already hashed, you must normalize it before hashing. For details, see [Normalization and Encoding](../getting-started/gs-normalization-encoding.md).
+Determine which method is best for you, and then follow the applicable integration guide.
 
-The UID2 module encrypts the hashed DII before sending it to the UID2 service.
+| Scenario | Integration Guide |
+| :--- | :--- |
+| You have access to DII on the client side and want to do front-end development only | [Prebid.js Client-Side Integration](integration-prebid-client-side.md) |
+| You have access to DII on the server side and can do server-side development | [Prebid.js Server-Side Integration](integration-prebid-server-side.md) |
 
-You can configure the module to send any one of the four accepted DII formats, for any specific user. The DII format might vary per user but you can only send one value per user.
+## Refreshing the UID2 Token
 
-The following sections demonstrate the different ways that you can configure the UID2 module and list the requirements for the DII passed to the module:
+The Prebid.js UID2 module can automatically refresh the UID2 tokens. If you prefer to implement manual refresh outside Prebid.js, see [Refreshing a UID2 Token](integration-prebid-server-side.md#refreshing-a-uid2-token) in the Server-Side Integration Guide.
+(**GWH_ query: is there no manual refresh option for client-side?**)
```

Comment:
yeh there is no manual refresh on client side

Reply:
@sunnywu thx. Slightly updated the wording here to include: " The client-side integration solution includes automated token refresh."